plugins/module_utils/nbdkit/nbdkit.go

```diff
@@ -48,6 +48,48 @@ type NbdkitServer struct {
 	socket string
 }
 
+func (c *NbdkitConfig) RunNbdKitFromLocal(diskName, diskPath string) (*NbdkitServer, error) {
+	path := fmt.Sprintf("/%s/%s", diskPath, diskName)
```

Comment:
```suggestion
	path := fmt.Sprintf("%s/%s", diskPath, diskName)
```

first `/` should be redundant since we have already a path here

Reply:
but isn't redundant the first `/` ? We should already inherit a `/foo/bar` path from `diskPath`, isn't it?